ccmlib/scylla_node.py

```diff
@@ -163,6 +164,13 @@ def get_cassandra_version(self):
         # TODO: Handle versioning
         return '3.0'
 
+    def has_native_nodetool(self):
+        if self._has_native_nodetool is None:
+            # TODO: handle enterprise versions
```

Comment:
why not handle it already ? we know what's the current version for enterprise branch

keep in mind ccm doesn't have enterprise branch, so it should be addressed here

Reply:
Ok, I'll handle it. We are still waiting on the scylla.git dependency anyway.

Reply:
Done in v4.